### PR TITLE
PR lesson_07

### PR DESCRIPTION
**Pull request by aageev2015**

### CODE REVIEW

**Comment by aageev2015** [comment]
3 days successfully completed
image

**Comment by aageev2015** [comment]
Raw folder+files created by job1
image

**Comment by aageev2015** [comment]
stg foolder+files created by job2
image

File: lesson_07/readme.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+Airflow learning. Windows + WSL
+
+Method (4) was used
+
+Methods:
+(1) Airflow via docker installation
+	Documentation: https://airflow.apache.org/docs/apache-airflow/stable/howto/docker-compose/index.html
```
Comment (v-hunt) [inline lesson_07/readme.md:7]: This method is cumbersome and too complicated :)

**Review by v-hunt — APPROVED** [review]
Excellent!

**Pull request merged**